src/main/java/frc/robot/subsystems/led/Leds.java

```diff
@@ -66,8 +71,22 @@ public enum LedMode {
   private static final double waveAllianceDuration = 2.0;
   private static final double autoFadeTime = 2.5; // 3s nominal
   private static final double autoFadeMaxTime = 5.0; // Return to normal
+  private static final double noteAngle = 5.0;
+
+  private NetworkTable ledTable;
+  private NetworkTableEntry ledModeEntry;
+  private NetworkTableEntry ledTestingEntry;
+  private NetworkTableEntry ledColor;
 
   public Leds() {
+    ledTable = NetworkTableInstance.getDefault().getTable("Leds");
+    ledModeEntry = ledTable.getEntry("Mode");
+    ledModeEntry.setString("OFF");
```

Comment:
It might be useful to be able to see which LED mode you're currently in.  Could you set ledModeEntry in updateLeds() when testing mode is not enabled?

Reply:
Actually in the else{...} in periodic() might be the best place to do this.

Reply:
done